libs/model/src/community/CreateGroup.command.ts

```diff
@@ -70,6 +70,22 @@ export function CreateGroup(): Command<
                 transaction,
               },
             );
+
+            // add topic level interaction permissions for current group
```

Comment:
Use `bulkCreate` instead of `Promise.all`

Reply:
i actually tried this and got a pretty weird enum error, it says its setting `varchar[]` into `enum_GroupPermissions_allowed_actions` which is pretty weird, any idea why it occurs?

Reply:
this seems to be related to the enum not being used in the model, but when you try the enum it breaks sequelize.sync with another weird error... probably a sequelize bug

We can leave it like this or change it to raw queries @timolegros ?

Reply:
This should work:
```
if (group.id) {
  // add topic level interaction permissions for current group
  const groupPermissions = (payload.topics || []).map((t) => ({
    group_id: group.id!,
    topic_id: t.id,
    allowed_actions: sequelize.literal(
      `ARRAY[${t.permissions.map((p) => `'${p}'`).join(', ')}]::"enum_GroupPermissions_allowed_actions"[]`,
    ) as unknown as PermissionEnum[],
  }));
  await models.GroupPermission.bulkCreate(groupPermissions, {
    transaction,
  });
}
 ```

Reply:
Thanks, updated @timolegros